examples/stable/postgresql/README.md

```diff
@@ -0,0 +1,248 @@
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters. This chart has been tested to work with NGINX Ingress, cert-manager, fluentd and Prometheus on top of the [BKPR](https://kubeprod.io/).
+
+## Prerequisites
+
+- Kubernetes 1.10+
+- PV provisioner support in the underlying infrastructure
+- Kanister version 0.21.0 installed
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+
+## Installing the Chart
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$ helm repo update
+
+$ helm install stable/postgresql --name my-release \
+	--namespace postgres-test \
+	--set image.repository=kanisterio/postgresql \
+	--set image.tag=0.21.0 \
+	--set postgresqlPassword=postgres-12345
+	--set postgresqlExtendedConf.archiveCommand="'envdir /bitnami/postgresql/data/env wal-e wal-push %p'" \
+	--set postgresqlExtendedConf.archiveMode=true \
+	--set postgresqlExtendedConf.archiveTimeout=60 \
+	--set postgresqlExtendedConf.walLevel=archive
+```
+
+The command deploys PostgreSQL on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+In case, if you don't have `Kanister` installed already, you can use following commands to do that.
+Add Kanister Helm repository and install Kanister operator
+```bash
+$ helm repo add kanister https://charts.kanister.io
+$ helm install --name kanister --namespace kasten-io kanister/kanister-operator --set image.tag=0.21.0
+```
+
+## Integrating with Kanister
+
+If you have deployed postgresql application with other name than `my-release` and namespace other than `postgres-test`, you need to modify the commands used below to use the correct name and namespace
+
+### Create Profile
+
+Create Profile CR if not created already
+
+```bash
+$ kanctl create profile s3compliant --access-key 'AKIAIOSFODNN7EXAMPLE' \
+	--secret-key 'wJalrXUtnFEMI%K7MDENG%bPxRfiCYEXAMPLEKEY' \
+	--bucket <s3-bucket-name> --region ap-south-1 \
+	--namespace postgres-test
+```
+
+**NOTE:**
+
+The command will configure a location where artifacts resulting from Kanister
+data operations such as backup should go. This is stored as a `profiles.cr.kanister.io`
+*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
+requires a Profile reference whether one created as part of the application install or
+not. Support for creating an ActionSet as part of install is simply for convenience.
+This CR can be shared between Kanister-enabled application instances so one option is to
+only create as part of the first instance.
```

Comment:
Lets change this to the following para since we are no longer installing profile as a part of application install:
```
The command will configure a location where artifacts resulting from Kanister
data operations such as backup should go. This is stored as a `profiles.cr.kanister.io`
*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
requires a Profile reference to complete the action. This CR (`profiles.cr.kanister.io`)
can be shared between Kanister-enabled application instances.
```